pg_cron.sql

```diff
@@ -1,6 +1,6 @@
 DO $$
 BEGIN
-   IF current_database() <> current_setting('cron.database_name') THEN
+   IF current_database() <> current_setting('cron.database_name') AND current_database() <> 'contrib_regression' THEN
```

Comment:
There's also a `REGRESS_OPTS = --dbname=postgres`  in the makefile. Do we still need this change? Or do we still need that line in the Makefile?

Reply:
Yes, this is needed. The ```REGRESS_OPTS``` is not necessary. I already removed it as you can see in the next commit https://github.com/citusdata/pg_cron/pull/47/commits/0305160449e4ea2b7f4862ad88b26f055d7fbc74.